src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2/Commands.DataFactoryV2.csproj

```diff
@@ -164,10 +164,13 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
     <Compile Include="Models\DataFactoryClient.Pipelines.cs" />
+    <None Include="AzureRM.DataFactoryV2.Netcore.psd1" />
     <None Include="AzureRM.DataFactoryV2.psd1">
       <Link>AzureRM.DataFactoryV2.psd1</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="Changelog.md" />
```

Comment:
Remove these lines as well

Reply:
Same here - it makes our development easier from VS